package.json

```diff
@@ -39,7 +39,7 @@
     "@types/puppeteer": "5.4.4",
     "@typescript-eslint/eslint-plugin": "^4.30.0",
     "@typescript-eslint/parser": "^4.0.0",
-    "@xrplf/eslint-config": "^1.2.2",
+    "@xrplf/eslint-config": "^1.3",
```

Comment:
Why is this showing up in this PR? This was an old PR right

Reply:
For some reason `2.0` is showing v1.2.2 as well? Must've not gotten merged into `2.0`

Reply:
Oh well, guess this PR will update that.